features/support/env.rb

```diff
@@ -1,5 +1,9 @@
 # Added for BrowserCMS (to test using test/dummy app)
 ENV["RAILS_ENV"] ||= "test"
+
+require "codeclimate-test-reporter"
+CodeClimate::TestReporter.start
```

Comment:
the test coverage does not work :( 

CodeClimate can only receive 1 input per commit, and this generates 2 (one for test, another for cucumber).

In any case, it's quite bad, ~80% or so.